com.unity.netcode.gameobjects/Runtime/Messaging/Messages/ConnectionApprovedMessage.cs

```diff
@@ -126,6 +148,12 @@ public void Handle(ref NetworkContext context)
             // Stop the client-side approval timeout coroutine since we are approved.
             networkManager.ConnectionManager.StopClientApprovalCoroutine();
 
+            networkManager.ConnectionManager.PeerClientIds.Clear();
```

Comment:
If there any danger of these lists getting multiple of the same peer in them?  From what I've seen it doesn't look possible, but I noticed with the NetcodeObject Observers you switched it to catch duplicates, is it worth taking the precaution on the peer list?

Reply:
Oh its cleared 😄 right there. Well no chance of that here but I would still like your thoughts.

Reply:
PeerClientIds is gone now. :) But it was also a HashSet, so it would have automatically handled duplicates.